### ISSUE TITLE
pnp telemetry

### ISSUE THREAD

**Issue by kmosti** [pnp/pnpjs#850]
### Category
- [ ] Enhancement
- [ ] Bug
- [ ] Question
- [X] Documentation gap/issue

### Version

Please specify what version of the library you are using: 1.3.3

Please specify what version(s) of SharePoint you are targeting: 2016

### Expected / Desired Behavior / Question
Why is there telemetry data being sent to https://pnptelemetryproxy.azurewebsites.net/track by default?

I could not find this documented anywhere, and I think this should obviously be more clearly communicated since you can glean confidential information just from the referrers (ie. site/Super-Secret-Project-To-Buy-A-Competitor.aspx).

I believe that this behavior should be controlled within e.g. the setup function in pnp/common, so it can be switched off.

**Comment by koltyakov** [comment]
No, we do not telemetry library usage by sending any additional requests to tracking endpoints. PnPjs contains custom header to identify the client so in SPO the requests from the library can be analyzed. In On-Prem no stats are collected at all.

**Comment by koltyakov** [comment]
The endpoint you mentioned is used in [@pnp/telemetry-js](https://github.com/pnp/telemetry-js) library, which is not a dependency in PnPjs. You probably face telemetry requests by using another project, e.g. [sp-dev-fx-controls-react](https://github.com/SharePoint/sp-dev-fx-controls-react) or [sp-dev-fx-property-controls](https://github.com/SharePoint/sp-dev-fx-property-controls).

Maybe @estruyf can help with more info about @pnp/telemetry-js and possibilities to turn it off on demand or point to the docs page with corresponding description.

**Comment by patrick-rodgers** [comment]
Transferring this as it does not relate to this library. We also do not use, as @koltyakov notes, the telemetry within this library.
